server/api.js

```diff
@@ -1,9 +1,87 @@
-import { Router } from "express";
+import { response, Router } from "express";
```

Comment:
What is this response? I didn't get and where are you using it.

Reply:
Whoops must have accidentally imported it. Usually happens when I miss type something and it autocorrects, such as the kebab incident 😂